### DISCUSSION TITLE
Obscurity fail for a closed-source app

### DISCUSSION THREAD

**Discussion by sarimrmalik** [Homebrew/discussions#2283] [Casks]
Hi, 

I submitted a new cask called `neat` to `homebrew-cask` and failed the "The Cask is for an app that is too obscure" test. 

The app is closed-source but I use a public github repo just to ship the releases. That's why it failed the test. 

Is there a way to bypass this test in this instance? Here is a [link to the PR](https://github.com/Homebrew/homebrew-cask/pull/112491) for reference.

**Comment by SMillerDev** [comment]
We have to base our judgement on something and if you are going to use the largest source distribution platform in the world, we are going to judge it with the metrics we get from them.
